pkg/resources/management.cattle.io/v3/setting/validator.go

```diff
@@ -129,3 +164,21 @@ func validateDuration(value string) (time.Duration, error) {
 
 	return dur, err
 }
+
+func clusterConditionMatches(cluster *v3.Cluster, t v3.ClusterConditionType, status v1.ConditionStatus) bool {
```

Comment:
I appreciate the tests on `effectiveValue`, I think for the sake of completeness we could add a test for `clusterConditionMatches` too

Reply:
I actually got rid of the tests of `effectiveValue`, as it will be tested by the exported `Admit` method - same with `clusterConditionMatches`.